daprdocs/content/en/developing-applications/building-blocks/secrets/secrets-overview.md

```diff
@@ -17,6 +17,9 @@ Using Dapr's secret store building block typically involves the following:
 1. Retrieving secrets using the Dapr secrets API in the application code.
 1. Optionally, referencing secrets in Dapr component files.
 
+Note: Dapr currently, by default, enables a built-in kubernetes secret store in kubernetes mode.
```

Comment:
```suggestion
{{% alert title="Note" color="primary" %}}
By default, Dapr currently enables a built-in Kubernetes secret store in Kubernetes mode. If you don't want to use any secrets API or would rather use a different secret store, you can disable the default Kubernetes secret store using the new `disable-builtin-k8s-secret-store` setting, which can be set to `true`. The default setting is `false`, to not disturb the ongoing default behavior of Dapr.
{{% /alert %}}
```

Encapsulating the whole note in shortcode

Reply:
I would suggest a slight variant like this.

```suggestion
Note: Dapr currently, by default, enables a built-in kubernetes secret store in kubernetes mode.
```

{{% alert title="Note" color="primary" %}}
By default, Dapr currently enables a built-in Kubernetes secret store in Kubernetes mode (deployed via the Helm defaults or `dapr init -k`). If you are using another secret store, you can disable the Dapr Kubernetes secret store using the `disable-builtin-k8s-secret-store` setting, when set to `true`. The default is `false`.
{{% /alert %}}

Reply:
Thanks @hhunter-ms  and @msfussell. Incorporated suggestions.